fbcp/init.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+if [ ! -e /dev/fb1 ]
+then
+    balena-idle
```

Comment:
We could print a message here to say the display wasn't detected? Alternatively we could exit the container instead of leaving it idling, and set restart on-failure in compose so it doesn't come back up.